addons/website_profile/static/src/components/profile_dialog/profile_dialog.js

```diff
@@ -0,0 +1,149 @@
+        useAutofocus({ refName: "name" });
+        const websiteDescriptionClass = this.getUniqueDomID("websiteDescription");
+        onWillStart(async () => {
+            const fields = [
+                "city",
+                "country_id",
+                "email",
+                "name",
+                "website",
+                "website_description",
+                "website_published",
+            ];
+            const [users, countries] = await Promise.all([
+                this.orm.read("res.users", [this.props.userId], fields),
+                this.orm.searchRead("res.country", [], ["id", "name"]),
+            ]);
+            this.user = reactive(users[0], () => this.validate());
+            this.user.country_id = this.user.country_id && this.user.country_id[0]; // keep only id
+            this.countries = countries;
+            const isInternalUser = await user.hasGroup("base.group_user");
+            this.descriptionConfig = {
+                allowFile: isInternalUser,
+                allowImage: isInternalUser,
+                classList: ["form-control", websiteDescriptionClass],
+                content: this.user.website_description,
+                debug: !!this.env.debug,
+                direction: localization.direction || "ltr",
+                placeholder: _t("Write a few words about yourself..."),
+            };
+        });
+        onMounted(() => {
+            if (this.props.focusWebsiteDescription) {
+                setTimeout(() => {
+                    const websiteDescription = document.querySelector(
+                        `.${websiteDescriptionClass}[contenteditable="true"]`
+                    );
+                    if (websiteDescription) {
+                        setCursorEnd(websiteDescription);
+                    }
+                }, 50);
+            }
+            this.validate();
+        });
+    }
+
+    get websiteDescription() {
+        const elContent = this.websiteDescriptionEditor.getElContent();
+        if (isHtmlEmpty(elContent.innerText)) {
+            return "";
+        }
+        return elContent.innerHTML;
+    }
+
+    get isEditingMyProfile() {
+        return user.userId === this.props.userId;
+    }
+
+    onEditProfileImg() {
+        this.upload.el.click();
+    }
+
+    onChangeProfileImg() {
+        const reader = new window.FileReader();
+        reader.addEventListener("load", (ev) => {
+            const result = ev.target.result;
+            this.profileImg.el.src = result;
+            this.profileImgData = result.toString().replace(/^data:.*;base64,/, "");
+        });
+        reader.readAsDataURL(this.upload.el.files[0]);
+    }
+
+    onClearProfileImg() {
+        this.profileImgData = false;
+        this.profileImg.el.src = "/web/static/img/placeholder.png";
+    }
+
+    onWebsiteDescriptionEditorLoad(editor) {
+        this.websiteDescriptionEditor = editor;
+    }
+
+    async _confirm() {
```

Comment:
Aren't you supposed to configure a "confirm" prop instead that links to your function?